safe_video/number_plate_recognition/utils.py

```diff
@@ -37,6 +39,20 @@ def merge_results(result1: Results, result2: Results) -> Results:
     merged_result.boxes.data = merged_data
     return merged_result
 
+def merge_results_list(results: list[Results]) -> Results:
```

Comment:
```suggestion
def merge_multiple_detection_results(results: list[Results]) -> Results:
```

Reply:
The other function `merge_results` also merges multiple detection results which makes this name not very helpful